CHANGES.rst

```diff
@@ -6,6 +6,7 @@ Version 8.3.x
 Unreleased
 
 -   Don't discard pager arguments by correctly using subprocess.Popen. :issue:`3039` :pr:`3055`
+-   Fix: Restore documented behavior for options with flag_value and is_flag=False (#3084).
```

Comment:
Follow the style in the rest of the changelog. Don't add a tag prefix, use the correct reference, add this to a new section rather than an already released version.